fence/blueprints/admin.py

```diff
@@ -495,6 +495,24 @@ def add_policy_to_user():
 
     return jsonify(res)
 
+@blueprint.route("/list_policies", methods=["GET"])
+@admin_login_required
+@debug_log
+def list_policies():
+    """
+    Return a list of all policies. Returns in JSON format
+    """
+    res = current_app.arborist.list_policies(False)
+    return jsonify(res)
+
+@blueprint.route("/list_policies?expand", methods=["GET"])
+@admin_login_required
+@debug_log
+def list_policies_expanded():
+    """Return a list of all policies with expanded information. Returns in JSON format """
+    res = current_app.arborist.list_policies(True)
+    return jsonify(res)
```

Comment:
Instead of making a separate endpoint please make this a parameter for the same endpoint.